tools/httpstan/README.md

```diff
@@ -0,0 +1,31 @@
+# bucketeer-httpstan
+
+Currently, the calculator service is written in Python, using Bayesian Algorithm to calculate the experiment
+probabilities.\
+Due to the following reasons, we want to rewrite the service in Go and, for Bayesian analysis, use the `HttpStan`.\
+This directory is used to build the `HttpStan` Docker image.
+
+## How to build httpstan image
+
+```bash
+make docker-build
```

Comment:
I tried to build the image on my machine, but I'm getting errors during this process.

https://github.com/bucketeer-io/bucketeer/pull/379/files#diff-2a88f9745b2764adf0e62db7d4d9f77d7430cc7efbf0d408d9ba2705cdc2f18cR15

```
➜  httpstan git:(feature/httpstan_image) make docker-build
docker build --platform linux/amd64 . -t bucketeer-httpstan:latest
[+] Building 104.1s (8/11)                                                                                                                                                                                                                                                                  
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                   0.0s
 => => transferring dockerfile: 44B                                                                                                                                                                                                                                                    0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                      0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                        0.0s
 => [internal] load metadata for docker.io/library/python:3.9-slim                                                                                                                                                                                                                     1.5s
 => [1/8] FROM docker.io/library/python:3.9-slim@sha256:5cde4e147c4165ad8dbf8a4df9631863766eeb0b79b890fafe6885b3b127af74                                                                                                                                                               0.0s
 => CACHED [2/8] WORKDIR /httpstan                                                                                                                                                                                                                                                     0.0s
 => CACHED [3/8] RUN apt-get update && apt-get install -y     build-essential     curl     software-properties-common     git     && rm -rf /var/lib/apt/lists/*                                                                                                                       0.0s
 => CACHED [4/8] RUN git clone https://github.com/stan-dev/httpstan.git .                                                                                                                                                                                                              0.0s
 => ERROR [5/8] RUN make                                                                                                                                                                                                                                                             102.5s
------                                                                                                                                                                                                                                                                                      
 > [5/8] RUN make:                                                                                                                                                                                                                                                                          
#8 0.422 downloading build/archives/math-v4.6.2.tar.gz                                                                                                                                                                                                                                      
#8 9.655 extracting archive build/archives/math-v4.6.2.tar.gz                                                                                                                                                                                                                               
#8 9.655 tar -C build -zxf build/archives/math-v4.6.2.tar.gz                                                                                                                                                                                                                                
#8 11.18 touch build/math-4.6.2                                                                                                                                                                                                                                                             
#8 11.19 make -f Makefile.libraries build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_nvecserial.a
#8 11.19 make[1]: Entering directory '/httpstan'
#8 11.26 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/nvector/serial/nvector_serial.c -o build/math-4.6.2/lib/sundials_6.1.1/src/nvector/serial/nvector_serial.o
#8 11.89 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.o
#8 11.93 ar -rs build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_nvecserial.a build/math-4.6.2/lib/sundials_6.1.1/src/nvector/serial/nvector_serial.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.o
#8 11.94 ar: creating build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_nvecserial.a
#8 11.94 make[1]: Leaving directory '/httpstan'
#8 11.94 mkdir -p httpstan/lib
#8 11.94 cp build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_nvecserial.a httpstan/lib/libsundials_nvecserial.a
#8 11.95 make -f Makefile.libraries build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_cvodes.a
#8 11.95 make[1]: Entering directory '/httpstan'
#8 12.01 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea.o
#8 12.47 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea_io.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea_io.o
#8 12.55 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes.o
#8 13.70 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bandpre.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bandpre.o
#8 13.82 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bbdpre.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bbdpre.o
#8 13.95 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_diag.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_diag.o
#8 14.02 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_direct.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_direct.o
#8 14.05 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_io.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_io.o
#8 14.30 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_ls.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_ls.o
#8 14.60 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls.o
#8 14.66 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_sim.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_sim.o
#8 14.74 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg.o
#8 14.81 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg1.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg1.o
#8 14.86 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_spils.c -o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_spils.o
#8 14.90 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_band.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_band.o
#8 15.01 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_context.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_context.o
#8 15.04 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_dense.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_dense.o
#8 15.27 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_direct.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_direct.o
#8 15.36 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_futils.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_futils.o
#8 15.38 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_iterative.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_iterative.o
#8 15.56 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_linearsolver.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_linearsolver.o
#8 15.60 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_matrix.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_matrix.o
#8 15.64 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_memory.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_memory.o
#8 15.68 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nonlinearsolver.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nonlinearsolver.o
#8 15.72 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector.o
#8 15.86 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector_senswrapper.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector_senswrapper.o
#8 15.97 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_version.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_version.o
#8 15.99 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/band/sunmatrix_band.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/band/sunmatrix_band.o
#8 16.11 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/dense/sunmatrix_dense.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/dense/sunmatrix_dense.o
#8 16.19 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/band/sunlinsol_band.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/band/sunlinsol_band.o
#8 16.25 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/dense/sunlinsol_dense.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/dense/sunlinsol_dense.o
#8 16.30 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/newton/sunnonlinsol_newton.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/newton/sunnonlinsol_newton.o
#8 16.36 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/fixedpoint/sunnonlinsol_fixedpoint.c -o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/fixedpoint/sunnonlinsol_fixedpoint.o
#8 16.50 ar -rs build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_cvodes.a build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodea_io.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bandpre.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_bbdpre.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_diag.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_io.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_ls.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_sim.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_nls_stg1.o build/math-4.6.2/lib/sundials_6.1.1/src/cvodes/cvodes_spils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_context.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_futils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_iterative.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_linearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_matrix.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_memory.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nonlinearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector_senswrapper.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_version.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/band/sunmatrix_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/dense/sunmatrix_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/band/sunlinsol_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/dense/sunlinsol_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/newton/sunnonlinsol_newton.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/fixedpoint/sunnonlinsol_fixedpoint.o
#8 16.51 ar: creating build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_cvodes.a
#8 16.51 make[1]: Leaving directory '/httpstan'
#8 16.51 mkdir -p httpstan/lib
#8 16.52 cp build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_cvodes.a httpstan/lib/libsundials_cvodes.a
#8 16.52 make -f Makefile.libraries build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_idas.a
#8 16.52 make[1]: Entering directory '/httpstan'
#8 16.58 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa.o
#8 17.00 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa_io.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa_io.o
#8 17.08 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas.o
#8 18.05 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_bbdpre.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_bbdpre.o
#8 18.17 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_direct.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_direct.o
#8 18.19 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ic.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ic.o
#8 18.42 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_io.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_io.o
#8 18.61 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ls.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ls.o
#8 18.88 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls.o
#8 18.94 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_sim.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_sim.o
#8 19.01 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_stg.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_stg.o
#8 19.07 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_spils.c -o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_spils.o
#8 19.11 ar -rs build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_idas.a build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idaa_io.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_bbdpre.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ic.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_io.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_ls.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_sim.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_nls_stg.o build/math-4.6.2/lib/sundials_6.1.1/src/idas/idas_spils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_context.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_futils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_iterative.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_linearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_matrix.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_memory.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nonlinearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector_senswrapper.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_version.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/band/sunmatrix_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/dense/sunmatrix_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/band/sunlinsol_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/dense/sunlinsol_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/newton/sunnonlinsol_newton.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/fixedpoint/sunnonlinsol_fixedpoint.o
#8 19.12 ar: creating build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_idas.a
#8 19.12 make[1]: Leaving directory '/httpstan'
#8 19.12 mkdir -p httpstan/lib
#8 19.13 cp build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_idas.a httpstan/lib/libsundials_idas.a
#8 19.13 make -f Makefile.libraries build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_kinsol.a
#8 19.13 make[1]: Entering directory '/httpstan'
#8 19.20 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol.o
#8 19.56 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_bbdpre.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_bbdpre.o
#8 19.66 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_direct.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_direct.o
#8 19.68 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_io.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_io.o
#8 19.78 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_ls.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_ls.o
#8 19.94 g++ -fPIC -pipe -pthread -D_REENTRANT  -O3 -I build/math-4.6.2/lib/sundials_6.1.1/include -I build/math-4.6.2/lib/sundials_6.1.1/src/sundials -DNO_FPRINTF_OUTPUT     -O3  -c -x c -include build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_spils.c -o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_spils.o
#8 19.97 ar -rs build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_kinsol.a build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol.o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_bbdpre.o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_io.o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_ls.o build/math-4.6.2/lib/sundials_6.1.1/src/kinsol/kinsol_spils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_context.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_direct.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_futils.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_iterative.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_linearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_math.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_matrix.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_memory.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nonlinearsolver.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_nvector_senswrapper.o build/math-4.6.2/lib/sundials_6.1.1/src/sundials/sundials_version.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/band/sunmatrix_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunmatrix/dense/sunmatrix_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/band/sunlinsol_band.o build/math-4.6.2/lib/sundials_6.1.1/src/sunlinsol/dense/sunlinsol_dense.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/newton/sunnonlinsol_newton.o build/math-4.6.2/lib/sundials_6.1.1/src/sunnonlinsol/fixedpoint/sunnonlinsol_fixedpoint.o
#8 19.97 ar: creating build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_kinsol.a
#8 19.98 make[1]: Leaving directory '/httpstan'
#8 19.98 mkdir -p httpstan/lib
#8 19.98 cp build/math-4.6.2/lib/sundials_6.1.1/lib/libsundials_kinsol.a httpstan/lib/libsundials_kinsol.a
#8 19.98 make -f Makefile.libraries build/math-4.6.2/lib/tbb/libtbb.so.2
#8 19.99 make[1]: Entering directory '/httpstan'
#8 20.05 touch build/math-4.6.2/lib/tbb/tbb-make-check
#8 20.06 tbb_root="../tbb_2020.3" CXX="g++" CC="gcc" LDFLAGS='-Wl,-L,"/httpstan/build/math-4.6.2/lib/tbb" -Wl,-rpath,"/httpstan/build/math-4.6.2/lib/tbb"  ' 'make' -C "build/math-4.6.2/lib/tbb" -r -f "/httpstan/build/math-4.6.2/lib/tbb_2020.3/build/Makefile.tbbmalloc" compiler=gcc cfg=release stdver=c++1y malloc CXXFLAGS="-Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation  "
#8 20.06 make[2]: Entering directory '/httpstan/build/math-4.6.2/lib/tbb'
#8 20.14 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/backend.cpp
#8 21.15 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/large_objects.cpp
#8 21.97 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/backref.cpp
#8 22.49 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/tbbmalloc.cpp
#8 22.94 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -o itt_notify_malloc.o -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/itt_notify.cpp
#8 23.63 sh ../tbb_2020.3/build/version_info_linux.sh g++ -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -Wall -Wextra -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1  -std=c++1y -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include >version_string.ver
#8 23.73 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm   -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1 -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc -I. ../tbb_2020.3/src/tbbmalloc/frontend.cpp
#8 24.76 echo "INPUT (libtbbmalloc.so.2)" > libtbbmalloc.so
#8 24.76 g++ -E -x c++ ../tbb_2020.3/src/tbbmalloc/lin64-tbbmalloc-export.def -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -Wall -Wextra -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1  -fno-rtti -fno-exceptions -D__TBBMALLOC_BUILD=1  -Wno-parentheses -Wno-sized-deallocation -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include > tbbmalloc.def
#8 24.77 gcc -fPIC -o libtbbmalloc.so.2 backend.o large_objects.o backref.o  tbbmalloc.o  itt_notify_malloc.o frontend.o  -ldl -lrt -shared -Wl,-soname=libtbbmalloc.so.2 -pthread -m64 -Wl,-L,"/httpstan/build/math-4.6.2/lib/tbb" -Wl,-rpath,"/httpstan/build/math-4.6.2/lib/tbb"   -Wl,--version-script,tbbmalloc.def
#8 24.81 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -Wall -Wextra -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1  -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -D__TBBMALLOC_BUILD=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/proxy.cpp
#8 25.30 g++ -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -Wall -Wextra -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1  -Wno-parentheses -Wno-sized-deallocation -fPIC -flifetime-dse=1 -D__TBBMALLOC_BUILD=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I../tbb_2020.3/src/tbbmalloc -I../tbb_2020.3/src/tbbmalloc ../tbb_2020.3/src/tbbmalloc/tbb_function_replacement.cpp
#8 25.32 echo "INPUT (libtbbmalloc_proxy.so.2)" > libtbbmalloc_proxy.so
#8 25.32 g++ -E -x c++ ../tbb_2020.3/src/tbbmalloc/lin64-proxy-export.def -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -Wall -Wextra -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1  -Wno-parentheses -Wno-sized-deallocation -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include > tbbmallocproxy.def
#8 25.33 g++ -fPIC -o libtbbmalloc_proxy.so.2 proxy.o tbb_function_replacement.o  -ldl -lrt libtbbmalloc.so -shared -Wl,-soname=libtbbmalloc_proxy.so.2 -pthread -m64 -Wl,-L,"/httpstan/build/math-4.6.2/lib/tbb" -Wl,-rpath,"/httpstan/build/math-4.6.2/lib/tbb"   -Wl,--version-script,tbbmallocproxy.def
#8 25.39 make[2]: Leaving directory '/httpstan/build/math-4.6.2/lib/tbb'
#8 25.39 touch build/math-4.6.2/lib/tbb/version_tbb_2020.3
#8 25.40 tbb_root="../tbb_2020.3" CXX="g++" CC="gcc" LDFLAGS='-Wl,-L,"/httpstan/build/math-4.6.2/lib/tbb" -Wl,-rpath,"/httpstan/build/math-4.6.2/lib/tbb"  ' 'make' -C "build/math-4.6.2/lib/tbb" -r -f "/httpstan/build/math-4.6.2/lib/tbb_2020.3/build/Makefile.tbb" compiler=gcc cfg=release stdver=c++1y  CXXFLAGS="-Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation  "
#8 25.40 make[2]: Entering directory '/httpstan/build/math-4.6.2/lib/tbb'
#8 25.42 /httpstan/build/math-4.6.2/lib/tbb_2020.3/build/Makefile.tbb:28: CONFIG: cfg=release arch=intel64 compiler=gcc target=linux runtime=cc10.2.1_libc2.31_kernel5.10.88
#8 25.43 g++ -o concurrent_hash_map.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/concurrent_hash_map.cpp
#8 26.00 g++ -o concurrent_queue.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/concurrent_queue.cpp
#8 26.86 g++ -o concurrent_vector.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/concurrent_vector.cpp
#8 27.94 g++ -o dynamic_link.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/dynamic_link.cpp
#8 28.48 g++ -o itt_notify.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/itt_notify.cpp
#8 29.17 g++ -o cache_aligned_allocator.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/cache_aligned_allocator.cpp
#8 29.77 g++ -o pipeline.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/pipeline.cpp
#8 30.47 g++ -o queuing_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/queuing_mutex.cpp
#8 31.04 g++ -o queuing_rw_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/queuing_rw_mutex.cpp
#8 31.68 g++ -o reader_writer_lock.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/reader_writer_lock.cpp
#8 32.33 g++ -o spin_rw_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/spin_rw_mutex.cpp
#8 32.82 g++ -o x86_rtm_rw_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/x86_rtm_rw_mutex.cpp
#8 33.39 g++ -o spin_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/spin_mutex.cpp
#8 33.95 g++ -o critical_section.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/critical_section.cpp
#8 34.63 g++ -o mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/mutex.cpp
#8 35.18 g++ -o recursive_mutex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/recursive_mutex.cpp
#8 35.65 g++ -o condition_variable.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/condition_variable.cpp
#8 36.32 g++ -o tbb_thread.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/tbb_thread.cpp
#8 36.89 g++ -o concurrent_monitor.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/concurrent_monitor.cpp
#8 37.51 g++ -o semaphore.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/semaphore.cpp
#8 37.94 g++ -o private_server.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/private_server.cpp
#8 38.65 g++ -o rml_tbb.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/rml/client/rml_tbb.cpp
#8 38.77 g++ -o tbb_misc.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include -I. ../tbb_2020.3/src/tbb/tbb_misc.cpp
#8 39.39 g++ -o tbb_misc_ex.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/tbb_misc_ex.cpp
#8 39.92 g++ -o task.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/task.cpp
#8 40.65 g++ -o task_group_context.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/task_group_context.cpp
#8 41.46 g++ -o governor.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/governor.cpp
#8 42.20 g++ -o market.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/market.cpp
#8 43.07 g++ -o arena.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/arena.cpp
#8 44.23 g++ -o scheduler.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/scheduler.cpp
#8 45.62 g++ -o observer_proxy.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/observer_proxy.cpp
#8 46.37 g++ -o tbb_statistics.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/tbb_statistics.cpp
#8 46.47 g++ -o tbb_main.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/tbb/tbb_main.cpp
#8 47.23 g++ -o concurrent_vector_v2.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/old/concurrent_vector_v2.cpp
#8 47.90 g++ -o concurrent_queue_v2.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/old/concurrent_queue_v2.cpp
#8 48.51 g++ -o spin_rw_mutex_v2.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/old/spin_rw_mutex_v2.cpp
#8 49.01 g++ -o task_v2.o -c -MMD -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -std=c++1y  -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include ../tbb_2020.3/src/old/task_v2.cpp
#8 49.49 sh ../tbb_2020.3/build/generate_tbbvars.sh
#8 49.49 echo "INPUT (libtbb.so.2)" > libtbb.so
#8 49.50 g++ -E -x c++ ../tbb_2020.3/src/tbb/lin64-tbb-export.def -O2 -g -DDO_ITT_NOTIFY -DUSE_PTHREAD -pthread -m64 -mrtm  -fPIC -flifetime-dse=1 -D__TBB_BUILD=1 -Wall -Wextra -Wno-parentheses -Wno-sized-deallocation -Wno-unknown-warning-option -Wno-deprecated-copy -Wno-missing-attributes -Wno-class-memaccess -Wno-sized-deallocation   -DTBB_SUPPRESS_DEPRECATED_MESSAGES=1 -I../tbb_2020.3/src -I../tbb_2020.3/src/rml/include -I../tbb_2020.3/include > tbb.def
#8 49.51 g++ -fPIC -o libtbb.so.2 concurrent_hash_map.o concurrent_queue.o concurrent_vector.o dynamic_link.o itt_notify.o cache_aligned_allocator.o pipeline.o queuing_mutex.o queuing_rw_mutex.o reader_writer_lock.o spin_rw_mutex.o x86_rtm_rw_mutex.o spin_mutex.o critical_section.o mutex.o recursive_mutex.o condition_variable.o tbb_thread.o concurrent_monitor.o semaphore.o private_server.o rml_tbb.o tbb_misc.o tbb_misc_ex.o task.o task_group_context.o governor.o market.o arena.o scheduler.o observer_proxy.o tbb_statistics.o tbb_main.o concurrent_vector_v2.o concurrent_queue_v2.o spin_rw_mutex_v2.o task_v2.o   -ldl -lrt -shared -Wl,-soname=libtbb.so.2 -pthread -m64 -Wl,-L,"/httpstan/build/math-4.6.2/lib/tbb" -Wl,-rpath,"/httpstan/build/math-4.6.2/lib/tbb"   -Wl,--version-script,tbb.def
#8 49.60 make[2]: Leaving directory '/httpstan/build/math-4.6.2/lib/tbb'
#8 49.60 make[1]: Leaving directory '/httpstan'
#8 49.60 cp build/math-4.6.2/lib/tbb/libtbb.so.2 httpstan/lib/libtbb.so.2
#8 49.61 cd httpstan/lib/ && cp libtbb.so.2 libtbb.so
#8 49.61 downloading build/archives/pybind11-2.9.2.tar.gz
#8 49.62 curl --silent --location https://github.com/pybind/pybind11/archive/v2.9.2.tar.gz -o build/archives/pybind11-2.9.2.tar.gz
#8 50.44 extracting archive build/archives/pybind11-2.9.2.tar.gz
#8 50.44 tar -C build -zxf build/archives/pybind11-2.9.2.tar.gz
#8 50.46 touch build/pybind11-2.9.2
#8 50.47 cp -r build/pybind11-2.9.2/include/pybind11 httpstan/include/pybind11
#8 50.47 downloading build/archives/rapidjson-1.1.0.tar.gz
#8 52.32 extracting archive build/archives/rapidjson-1.1.0.tar.gz
#8 52.32 tar -C build -zxf build/archives/rapidjson-1.1.0.tar.gz
#8 52.35 touch build/rapidjson-1.1.0
#8 52.36 cp -r build/rapidjson-1.1.0/include/rapidjson httpstan/include/rapidjson
#8 52.36 downloading build/archives/stan-v2.32.2.tar.gz
#8 54.73 extracting archive build/archives/stan-v2.32.2.tar.gz
#8 54.73 tar -C build -zxf build/archives/stan-v2.32.2.tar.gz
#8 54.83 touch build/stan-2.32.2
#8 54.83 cp -r build/stan-2.32.2/src/stan httpstan/include/stan
#8 54.84 cp build/math-4.6.2/stan/math.hpp httpstan/include/stan
#8 54.84 cp -r build/math-4.6.2/stan/math httpstan/include/stan
#8 54.90 cp -r build/math-4.6.2/lib/boost_1.78.0/boost httpstan/include/boost
#8 55.37 cp -r build/math-4.6.2/lib/eigen_3.4.0/Eigen build/math-4.6.2/lib/eigen_3.4.0/unsupported httpstan/include
#8 55.40 cp -r build/math-4.6.2/lib/sundials_6.1.1/include/cvodes build/math-4.6.2/lib/sundials_6.1.1/include/idas build/math-4.6.2/lib/sundials_6.1.1/include/kinsol build/math-4.6.2/lib/sundials_6.1.1/include/nvector build/math-4.6.2/lib/sundials_6.1.1/include/sundials build/math-4.6.2/lib/sundials_6.1.1/include/sunlinsol build/math-4.6.2/lib/sundials_6.1.1/include/sunmatrix build/math-4.6.2/lib/sundials_6.1.1/include/sunmemory build/math-4.6.2/lib/sundials_6.1.1/include/sunnonlinsol build/math-4.6.2/lib/sundials_6.1.1/include/stan_sundials_printf_override.hpp build/math-4.6.2/lib/sundials_6.1.1/include/sundials_debug.h httpstan/include
#8 55.40 cp -r build/math-4.6.2/lib/tbb_2020.3/include/tbb httpstan/include
#8 55.41 curl --location https://github.com/stan-dev/stanc3/releases/download/v2.32.2/linux-stanc -o build/stanc --retry 5 --fail
#8 55.42   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
#8 55.42                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13.4M  100 13.4M    0     0  3256k      0  0:00:04  0:00:04 --:--:-- 3979k
#8 59.66 rm -f httpstan/stanc && cp -r build/stanc httpstan/stanc && chmod u+x httpstan/stanc
#8 59.79 # -fvisibility=hidden required by pybind11
#8 59.79 g++ \
#8 59.79 	-Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall \
#8 59.79 	-fPIC \
#8 59.79 	-DBOOST_DISABLE_ASSERTS -DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION -DSTAN_THREADS -D_REENTRANT -D_GLIBCXX_USE_CXX11_ABI=0 \
#8 59.79 	-Ihttpstan -Ihttpstan/include \
#8 59.79 	-I/usr/local/include/python3.9 \
#8 59.79 	-I/usr/local/include/python3.9 \
#8 59.79 	-fvisibility=hidden \
#8 59.79 	-c httpstan/stan_services.cpp -o httpstan/stan_services.o \
#8 59.79 	-O3 -std=c++14
#8 69.12 In file included from httpstan/include/stan/math/prim/functor.hpp:12,
#8 69.12                  from httpstan/include/stan/math/rev/fun.hpp:198,
#8 69.12                  from httpstan/include/stan/math/rev.hpp:10,
#8 69.12                  from httpstan/include/stan/math.hpp:19,
#8 69.12                  from httpstan/include/stan/io/array_var_context.hpp:6,
#8 69.12                  from httpstan/stan_services.cpp:8:
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp: In function ‘void stan::math::internal::combination(std::vector<int>&, const int&, const int&, const int&)’:
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp:73:29: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
#8 69.12    73 |   for (std::size_t i = 0; i < p - 1; i++) {
#8 69.12       |                           ~~^~~~~~~
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp:79:16: warning: comparison of integer expressions of different signedness: ‘size_t’ {aka ‘long unsigned int’} and ‘const int’ [-Wsign-compare]
#8 69.12    79 |     } while (k < x);
#8 69.12       |              ~~^~~
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp: In function ‘void stan::math::internal::combos(const int&, const double&, const int&, std::vector<std::vector<double> >&)’:
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp:102:29: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
#8 69.12   102 |   for (std::size_t i = 1; i != choose_dimk + 1; i++) {
#8 69.12       |                           ~~^~~~~~~~~~~~~~~~~~
#8 69.12 httpstan/include/stan/math/prim/functor/hcubature.hpp:105:31: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘const int’ [-Wsign-compare]
#8 69.12   105 |     for (std::size_t j = 0; j != k; j++) {
#8 69.12       |                             ~~^~~~
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp: In function ‘void stan::math::internal::increment(std::vector<bool>&, const int&, const double&, const std::vector<int>&, std::vector<double>&)’:
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp:126:31: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘const int’ [-Wsign-compare]
#8 69.13   126 |     for (std::size_t j = 0; j != k; j++) {
#8 69.13       |                             ~~^~~~
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp:132:22: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<bool>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 69.13   132 |   while ((first_zero < index.size()) && index[first_zero]) {
#8 69.13       |           ~~~~~~~~~~~^~~~~~~~~~~~~~
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp:135:18: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<bool>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 69.13   135 |   if (first_zero == index.size()) {
#8 69.13       |       ~~~~~~~~~~~^~~~~~~~~~~~~~~
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp:143:31: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
#8 69.13   143 |     for (std::size_t i = 0; i != first_zero + 1; i++) {
#8 69.13       |                             ~~^~~~~~~~~~~~~~~~~
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp: In function ‘void stan::math::internal::signcombos(const int&, const double&, const int&, std::vector<std::vector<double> >&)’:
#8 69.13 httpstan/include/stan/math/prim/functor/hcubature.hpp:168:29: warning: comparison of integer expressions of different signedness: ‘std::size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
#8 69.13   168 |   for (std::size_t i = 1; i != choose_dimk + 1; i++) {
#8 69.13       |                           ~~^~~~~~~~~~~~~~~~~~
#8 72.47 In file included from httpstan/stan_services.cpp:8:
#8 72.47 httpstan/include/stan/io/array_var_context.hpp: In member function ‘virtual std::vector<std::complex<double> > stan::io::array_var_context::vals_c(const string&) const’:
#8 72.47 httpstan/include/stan/io/array_var_context.hpp:304:52: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<double>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 72.47   304 |       for (comp_iter = 0, real_iter = 0; real_iter < val_r->second.first.size();
#8 72.47       |                                          ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#8 72.47 httpstan/include/stan/io/array_var_context.hpp:317:24: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<int>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 72.47   317 |              real_iter < val_i->second.first.size();
#8 72.47       |              ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#8 72.50 In file included from httpstan/include/stan/services/util/initialize.hpp:7,
#8 72.50                  from httpstan/include/stan/services/sample/fixed_param.hpp:13,
#8 72.50                  from httpstan/stan_services.cpp:11:
#8 72.50 httpstan/include/stan/io/random_var_context.hpp: In member function ‘virtual std::vector<std::complex<double> > stan::io::random_var_context::vals_c(const string&) const’:
#8 72.50 httpstan/include/stan/io/random_var_context.hpp:111:52: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<double>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 72.50   111 |       for (comp_iter = 0, real_iter = 0; real_iter < val_r.size();
#8 72.50       |                                          ~~~~~~~~~~^~~~~~~~~~~~~~
#8 72.66 In file included from httpstan/include/stan/services/util/create_unit_e_diag_inv_metric.hpp:4,
#8 72.66                  from httpstan/include/stan/services/util/inv_metric.hpp:1,
#8 72.66                  from httpstan/include/stan/services/sample/hmc_nuts_diag_e_adapt.hpp:14,
#8 72.66                  from httpstan/stan_services.cpp:12:
#8 72.66 httpstan/include/stan/io/dump.hpp: In member function ‘virtual std::vector<std::complex<double> > stan::io::dump::vals_c(const string&) const’:
#8 72.66 httpstan/include/stan/io/dump.hpp:694:52: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<double>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 72.66   694 |       for (comp_iter = 0, real_iter = 0; real_iter < val_r->second.first.size();
#8 72.66       |                                          ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#8 72.66 httpstan/include/stan/io/dump.hpp:707:24: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<int>::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 72.66   707 |              real_iter < val_i->second.first.size();
#8 72.66       |              ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#8 74.38 In file included from httpstan/include/stan/math/prim/fun.hpp:124,
#8 74.38                  from httpstan/include/stan/math/rev/fun/multiply.hpp:7,
#8 74.38                  from httpstan/include/stan/math/rev/fun/elt_multiply.hpp:9,
#8 74.38                  from httpstan/include/stan/math/rev/fun.hpp:55,
#8 74.38                  from httpstan/include/stan/math/rev.hpp:10,
#8 74.38                  from httpstan/include/stan/math.hpp:19,
#8 74.38                  from httpstan/include/stan/io/array_var_context.hpp:6,
#8 74.38                  from httpstan/stan_services.cpp:8:
#8 74.38 httpstan/include/stan/math/prim/fun/grad_2F1.hpp: In instantiation of ‘TupleT stan::math::internal::grad_2F1_impl(const T1&, const T2&, const T3&, const T_z&, double, int) [with bool calc_a1 = true; bool calc_a2 = true; bool calc_b1 = true; bool calc_z = true; T1 = double; T2 = double; T3 = double; T_z = double; ScalarT = double; TupleT = std::tuple<double, double, double, double>]’:
#8 74.38 httpstan/include/stan/math/prim/fun/grad_2F1.hpp:307:57:   required from ‘auto stan::math::grad_2F1(const T1&, const T2&, const T3&, const T_z&, double, int) [with bool ReturnSameT = true; T1 = double; T2 = double; T3 = double; T_z = double; stan::require_t<std::integral_constant<bool, __v> >* <anonymous> = 0]’
#8 74.38 httpstan/include/stan/math/prim/fun/grad_inc_beta.hpp:37:46:   required from here
#8 74.38 httpstan/include/stan/math/prim/fun/grad_2F1.hpp:192:12: warning: unused variable ‘pre_mult’ [-Wunused-variable]
#8 74.38   192 |       auto pre_mult = a2 * pow(1 - z, -1 - a2);
#8 74.38       |            ^~~~~~~~
#8 74.44 httpstan/include/stan/math/prim/fun/grad_2F1.hpp: In instantiation of ‘TupleT stan::math::internal::grad_2F1_impl(const T1&, const T2&, const T3&, const T_z&, double, int) [with bool calc_a1 = true; bool calc_a2 = true; bool calc_b1 = true; bool calc_z = true; T1 = stan::math::var_value<double>; T2 = stan::math::var_value<double>; T3 = stan::math::var_value<double>; T_z = stan::math::var_value<double>; ScalarT = stan::math::var_value<double>; TupleT = std::tuple<stan::math::var_value<double, void>, stan::math::var_value<double, void>, stan::math::var_value<double, void>, stan::math::var_value<double, void> >]’:
#8 74.44 httpstan/include/stan/math/prim/fun/grad_2F1.hpp:307:57:   required from ‘auto stan::math::grad_2F1(const T1&, const T2&, const T3&, const T_z&, double, int) [with bool ReturnSameT = true; T1 = stan::math::var_value<double>; T2 = stan::math::var_value<double>; T3 = stan::math::var_value<double>; T_z = stan::math::var_value<double>; stan::require_t<std::integral_constant<bool, __v> >* <anonymous> = 0]’
#8 74.44 httpstan/include/stan/math/rev/fun/grad_inc_beta.hpp:49:51:   required from here
#8 74.44 httpstan/include/stan/math/prim/fun/grad_2F1.hpp:192:12: warning: variable ‘pre_mult’ set but not used [-Wunused-but-set-variable]
#8 74.83 In file included from httpstan/stan_services.cpp:8:
#8 74.83 httpstan/include/stan/io/array_var_context.hpp: In instantiation of ‘std::vector<long unsigned int> stan::io::array_var_context::validate_dims(const std::vector<std::basic_string<char> >&, T, const std::vector<std::vector<long unsigned int> >&) [with T = long unsigned int]’:
#8 74.83 httpstan/include/stan/io/array_var_context.hpp:97:75:   required from here
#8 74.83 httpstan/include/stan/io/array_var_context.hpp:74:23: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<std::vector<long unsigned int> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 74.83    74 |     for (int i = 0; i < dims.size(); i++) {
#8 74.83       |                     ~~^~~~~~~~~~~~~
#8 74.84 httpstan/include/stan/io/array_var_context.hpp: In instantiation of ‘std::vector<long unsigned int> stan::io::array_var_context::validate_dims(const std::vector<std::basic_string<char> >&, T, const std::vector<std::vector<long unsigned int> >&) [with T = long int]’:
#8 74.84 httpstan/include/stan/io/array_var_context.hpp:118:75:   required from here
#8 74.84 httpstan/include/stan/io/array_var_context.hpp:74:23: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<std::vector<long unsigned int> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
#8 102.5 g++: fatal error: Killed signal terminated program cc1plus
#8 102.5 compilation terminated.
#8 102.5 make: *** [Makefile:226: httpstan/stan_services.o] Error 1
```

Reply:
I have change the Dockerfile `python:3.9-slim` to `python:3.9` , and I tried agian on my machine, it show going well: \

![image](https://github.com/bucketeer-io/bucketeer/assets/34117229/34cabbcc-4402-4d75-9916-0af9f23b37d5)

you can update this branch and try again.

Reply:
Before you try again, you can use this command to clean docker builder cache: `docker builder prune`

Reply:
I'm still getting the same error.
Can you share your machine environment? OS and CPU?

Reply:
> I'm still getting the same error. Can you share your machine environment? OS and CPU?
Here is my machine info:
OS: MacOS Monterey 13.4
CPU: Inter i5-1038NG7
<img width="507" alt="image" src="https://github.com/bucketeer-io/bucketeer/assets/34117229/7f66e2fa-8efd-4fdd-b2ab-c372772ac5ec">

Reply:
The issue was the memory.
Once I increased the VM memory, it worked. M1 and Intel CPUs were also checked. No issues.

Can you add this text to the README?

```
## How to build httpstan image

If you have compiling errors during the image build process, you might need to increase the docker VM memory.
``` 

Also, please change back the Python image to `slim`.

Reply:
No problem.